invenio_oauthclient/alembic/97bbc733896c_create_oauthclient_tables.py

```diff
@@ -73,6 +73,7 @@ def upgrade():
         sa.Column('secret', sa.Text(), nullable=False),
         sa.ForeignKeyConstraint(
             ['id_remote_account'], [u'oauthclient_remoteaccount.id'],
+            name='fk_oauthclient_remote_token_remote_account'
```

Comment:
I as understand the upgrade is actually broken as it is today since it will never actually create the table on e.g. MySQL because the naming conventions where introduced after this upgrade was merged? I.e. there won't be a table on MySQL to upgrade.